pkg/frontend/openshiftcluster_putorpatch_test.go

```diff
@@ -141,8 +300,6 @@ func TestPutOrPatchOpenShiftClusterAdminAPI(t *testing.T) {
 		{
 			name: "patch a cluster with registry profile should ignore registry profile",
 			request: func(oc *admin.OpenShiftCluster) {
-				oc.Properties.ClusterProfile.Domain = "changed"
-				oc.Name = "resourceName"
```

Comment:
I don't follow why we change this

Reply:
See https://github.com/Azure/ARO-RP/pull/1803